src/main/java/com/prefect/office/record/management/app/facade/prefect/violation/impl/ViolationFacadeImpl.java

```diff
@@ -2,11 +2,24 @@
 
 import com.prefect.office.record.management.app.facade.prefect.violation.ViolationFacade;
 import com.prefect.office.record.management.app.model.violation.Violation;
+import com.prefect.office.record.management.data.connectionhelper.ConnectionHelper;
 import com.prefect.office.record.management.data.dao.prefect.violation.ViolationDao;
 import com.prefect.office.record.management.data.dao.prefect.violation.impl.ViolationDaoImpl;
+import oracle.jdbc.proxy.annotation.Pre;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.List;
 
 
 public class ViolationFacadeImpl implements ViolationFacade {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ViolationFacade.class);
+    Connection c = ConnectionHelper.getConnection();
```

Comment:
Do not make global connection. this has to be closed everytime it gets open. either:
1. localize the use of connection the close it afterwards
or 
2. use try with resources